test/fixtures/project/.psysh.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Created a separate directory for this file so it clearly for a different test than directories.